code/game/objects/items/weapons/teleportation.dm

```diff
@@ -92,15 +92,17 @@ Frequency:
 					var/turf/tr = get_turf(W)
 					if (tr.z == sr.z && tr)
 						var/direct = max(abs(tr.x - sr.x), abs(tr.y - sr.y))
-						if (direct < 20)
-							if (direct < 5)
-								direct = "very strong"
+						if (direct < 5)
```

Comment:
Could you change this to use a switch statement? Like so:

```dm
switch (direct)
    if (0 to 5)
        direct = "very strong"
    if (5 to 10)
        direct = "strong"
    if (10 to 20)
        direct = "weak"
    else
        direct = "very weak"
```

Reply:
Done and done!  I like it.